_rules/table-headers-attribute-refer-to-data-cells.md

```diff
@@ -0,0 +1,223 @@
+---
+id: a25f45
+name: `headers` attribute only refers to `cells` in the same `table` element
+test_type: atomic
+description: |
+  This rule checks that the `headers` attribute must only refer to `cells` in the same `table` element
+test_aspects:
+- DOM Tree
+authors:
+- Jey Nandakumar
+- Audrey Maniez
+---
+
+## Test Procedure
+
+### Applicability
+
+This rule applies to `td` and `th` elements with a `headers` attribute and within a `table` element with a [semantic role](#semantic-role) of `table` or `grid` or `treegrid`, that is [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+### Expectation
+
+Each target element with the `headers` attribute refers to other `th` of the same `table` element.
+
+## Assumptions
+
+- This test assumes that the `headers` attribute is only used to identify table headers. If other information is included in the headers attribute, the rule may fail on issues that are not accessibility concerns. For example, if `headers` is used to include information for script, this rule may not be accurate.
+
+## Accessibility support
+
+- `headers` attribute may not be consistently announced by assistive technologies.
+
+## Background
+
+- [Understanding Success Criterion 1.3.1: Information and relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
+- [H43: Using id and headers attributes to associate data cells with header cells in data tables](https://www.w3.org/WAI/WCAG21/Techniques/html/H43)
+- [Failure of Success Criterion 1.3.1 for incorrectly associating table headers and content via the headers and id attributes](https://www.w3.org/WAI/WCAG21/Techniques/failures/F90)
```

Comment:
I think we should identify the failure by its number (as it is done with the technic above ?) 

> - [F90: Incorrectly associating table headers and content via the headers and id attributes](https://www.w3.org/WAI/WCAG21/Techniques/failures/F90)